ratis-docs/src/site/markdown/index.md

```diff
@@ -17,12 +17,27 @@
 
 # Apache Ratis
 Apache Ratis is a highly customizable Raft protocol implementation in Java.
-Raft is a easily understandable consensus algorithm to manage replicated state.
-Apache Ratis could be used in any Java application where state should be replicated between multiple instances.
+[Raft](https://raft.github.io/) is an easily understandable consensus algorithm to manage replicated state. 
+
+The Ratis project was started at 2017 as an incubator project, and has graduated at Feb 17, 2021. Originally, Ratis is built for using Raft in [Apache Ozone](https://ozone.apache.org) where Ratis is used to replicate raw data and to provide high availability, and the correctness and performance of Ratis has been heavily tested with Ozone. 
```

Comment:
Revised as below:
```
The Ratis project was started at 2016,
entered Apache incubation in 2017,
and graduated as a top level Apache project on Feb 17, 2021.
Originally, Ratis was built for using Raft in [Apache Ozone](https://ozone.apache.org)
in order to replicate raw data and to provide high availability.
The correctness and performance of Ratis has been heavily tested with Ozone.
```

BTW, let's break the long lines to multiple short lines so that it is easier to provide comments.

Reply:
@szetszwo Thanks for reviewing! Already updated.